client/components/masterbar-styled/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Global, css } from '@emotion/react';
```

Comment:
This is a copy of https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/checkout-thank-you/marketplace/masterbar-styled/index.tsx